### PR TITLE
Fix empty SFSafariViewController bug

### PR DESCRIPTION
**Pull request by joewalsh**
- Use namespace returned by search to prevent pushing non-articles to the article vc
- Fetch summary response for externally loaded articleURLs to verify they're articles before pushing them to the article vc
- Remove external url fallback from article vc

https://phabricator.wikimedia.org/T214690

### CODE REVIEW

File: Wikipedia/Code/WMFAppViewController.m [hunk 1]
```diff
@@ -1202,8 +1201,28 @@ - (BOOL)processUserActivity:(NSUserActivity *)activity animated:(BOOL)animated c
...
+                    NSNumber *namespaceId = namespaceDictionary[@"id"];
+                    if (![namespaceId isKindOfClass:[NSNumber class]]) {
+                        bail();
+                        return;
+                    }
+                    if ([namespaceId integerValue] != 0) {
+                        bail();
+                        return;
+                    }
+                    [self showArticleForURL:URL animated:animated completion:done];
```
Comment (nambatee) [inline Wikipedia/Code/WMFAppViewController.m]: wouldn't you want to `bail` if there's an error?

Comment (joewalsh) [reply Wikipedia/Code/WMFAppViewController.m]: When there's an error, `response` will be nil and it'll bail at the first check

Comment (joewalsh) [reply Wikipedia/Code/WMFAppViewController.m]: @nambatee I thought about this some more, the app should show the error in this case. If it's a network error or some other error, pushing into the SFSafariVC would just be confusing. I added this change.

**Comment by joewalsh** [comment]
Holding pending more product discussion

**Comment by joewalsh** [comment]
Tagging into 6.2.1 due to impact

**Pull request closed without merging**